TACC/protx
# Task/COOKS-98: top navigation refactor and other demo requested changes

## Overview: ##

This PR refactors the top navigation (and adds more related routes) refactor and other requested changes for upcoming demo:

- Create 3 routes to be used by for the  _Demographics_, _Child Maltreatment_, _Analytics_ CMS links (i.e. refactor routing so that it handles mapType)
- Reorder maltreatment types and drop NA type 
- Remove "estimated" from any demographic labels
- Rename 'Feature' to 'Demographic'
- Rename 'TimeFrame' to 'Years'

## Related Jira tickets: ##

* [COOKS-98](https://jira.tacc.utexas.edu/browse/COOKS-98)

## Testing Steps: ##
1. Try all routes
2. See that labels have been changed as described in this PR
